django/library/models.py

```diff
@@ -2001,7 +2002,10 @@ def get_change_closed_url(self):
         return reverse("library:peer-review-change-closed", kwargs={"slug": self.slug})
 
     def get_invite(self, member_profile):
-        return self.invitation_set.filter(candidate_reviewer=member_profile).first()
+        # return self.invitation_set.filter(candidate_reviewer=member_profile).first()
+        return self.invitation_set.filter(
```

Comment:
should this be ordered by most recent e.g., `self.invitation_set.order_by('-date_created').filter(...).first()`

could also add a default `ordering` to `PeerReviewInvation` class Meta